src/auth/screens/events.screen.js

```diff
@@ -1,30 +1,20 @@
 /* eslint react/prop-types: 0 */
-
+/* eslint-disable no-shadow */
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
 import { StyleSheet, Text, FlatList, View } from 'react-native';
-import moment from 'moment';
+import moment from 'moment/min/moment-with-locales.min';
+import I18n from 'react-native-i18n';
 
 import { LoadingUserListItem, UserListItem, ViewContainer } from 'components';
 import { colors, fonts, normalize } from 'config';
 import { emojifyText, translate } from 'utils';
 import { getUserEvents } from '../auth.action';
 
-moment.updateLocale('en', {
-  relativeTime: {
-    past: '%s',
-    s: '%ds',
-    m: '%dm',
-    mm: '%dm',
-    h: '%dh',
-    hh: '%dh',
-    d: '%dd',
-    dd: '%dd',
-    M: '%dmo',
-    MM: '%dmo',
-    y: '%dy',
-    yy: '%dy',
-  },
+const language = I18n.locale.substr(0, 2);
```

Comment:
I have done so for the time being, but this is essentially the initialization of the `moment`, and it should not be executed here (where can it be moved?), it first. And secondly, only the device language is taken into account here, but if the language was changed in the settings, then the date/time language (ie `moment`) will remain unchanged (we must also take into account the language in the storage). What can we do about it?